src/core/dataset.ts

```diff
@@ -4,9 +4,20 @@ module Plottable {
 
 export type DatasetCallback = (dataset: Dataset) => void;
 
+export class KeyFunctions {
+  protected static counter: number = 0;
+  public static noConstancy: (d: any, i: number) => any = (d: any, i: number) => {
+    return KeyFunctions.counter++;
+  };
```

Comment:
Just curious, what's the difference between this and not using a key function at all?